scripts/plugins/storage/sqlite/upgrade/9.sql

```diff
@@ -0,0 +1,8 @@
+-- Insert into category_children
```

Comment:
These fixed names should not be used, we are moving away from them with the introduction of the add service and the and task mechanism. These calls should be the way in which these entries get populated.